doc/api/synopsis.markdown

```diff
@@ -1,29 +1,43 @@
-# Synopsis
+# Usage
```

Comment:
The title being different from the filename is potentially concerning.

Could the filename also be changed, and then a redirect added in `synopsis.html` to preserve links?